docs/gno-infrastructure/setting-up-a-faucet.md

```diff
@@ -0,0 +1,263 @@
+---
+id: setting-up-a-faucet
+---
+
+# Setting up a faucet for your Gno network
+
+In this tutorial, we will cover how to run a local native currency faucet that 
+works seamlessly with a Gno node. Using the faucet, any address can get a hold
+of testnet GNOTs. The faucet used in this tutorial can be found in
+[this repository](https://github.com/gnolang/faucet).
+
+## Prerequisites
+
+- Git
+- Go 1.21+
+- Make (for running Makefiles)
+- `gnoland` & `gnokey` installed
+- A `gnoland` node running
+- A Gno.land keypair generated using [`gnokey`](../gno-tooling/cli/gnokey.md) 
+containing funds for the faucet to serve
```

Comment:
These are 2 separate requirements -- please make them clear.

Also, please link some kind of guide on how to actually prefund an account